spec/classes/foreman_proxy__config__spec.rb

```diff
@@ -21,6 +28,7 @@
         usr_dir = '/usr'
         var_dir = '/var/lib/puppet'
       end
+      puppet_etc_dir = facts[:osfamily] == 'Archlinux' ? var_dir : "#{etc_dir}/puppet"
```

Comment:
Ok, i've renamed var_dir to ssl_dir, since that's all it's being used for, and then ssl_dir = /etc... looks a lot less weird in Arch...